projects/services/consensus-service-typescript/src/consensus-server.ts

```diff
@@ -1,21 +1,38 @@
-import { grpcServer, types, Config, topologyPeers, logger } from "orbs-core-library";
+import { defaults } from "lodash";
+
+import { grpcServer, types, topologyPeers, logger } from "orbs-core-library";
 import { Consensus, SubscriptionManager, TransactionPool } from "orbs-core-library";
 
 import ConsensusService from "./consensus-service";
 import SubscriptionManagerService from "./subscription-manager-service";
 import TransactionPoolService from "./transaction-pool-service";
 
-function makeConsensus(config: Config, peers: types.ClientMap) {
-  const consensusConfig = config.get("consensus");
-  if (!consensusConfig) {
-    throw new Error("Couldn't find consensus configuration!");
-  }
+const DEFAULT_CONSENSUS_CONFIG = {
+  electionTimeout: {
+    min: 2000,
+    max: 4000
+  },
+  heartbeatInterval: 100,
+};
+
+const { NODE_NAME, NUM_OF_NODES, ETHEREUM_CONTRACT_ADDRESS } = process.env;
```

Comment:
We better switch to nconf and let it parse the config params automatically. Also, this way, we can conveniently pass them as command line arguments if needed